pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/bestpractices/xml/JUnit5TestShouldBePackagePrivate.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<test-data
+    xmlns="http://pmd.sourceforge.net/rule-tests"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pmd.sourceforge.net/rule-tests http://pmd.sourceforge.net/rule-tests_1_0_0.xsd">
+
+    <test-code>
+        <description>Public modifier is not necessary on a test method nor on the class</description>
+        <expected-problems>2</expected-problems>
```

Comment:
with multiple problem, we can pin it down to a specific line-number to ensure, we don't flag something wrong.
here it would be

```xml
<expected-linenumbers>3,5</expected-linenumbers>
```

The expected-linenumbers needs to be added additionally to expected-problems.